package.json

```diff
@@ -4,7 +4,7 @@
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
-    "lint": "eslint .",
+    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
```

Comment:
No I mean what it is right now will work (this change is unnecessary). I am certain, I tried this in lots of different configurations.

Plus: https://docs.npmjs.com/misc/scripts#path